phpstan.neon

```diff
@@ -11,10 +11,7 @@ parameters:
 		- src/
 		- plugin.php
 
-#    ignoreErrors:
-#        - '#PHPDoc tag @var#'
-#
-#    excludePaths:
-#        - ./*/*/FileToBeExcluded.php
-#
-#    checkMissingIterableValueType: false
+	scanDirectories:
+		- %currentWorkingDirectory%/../wordpress-fieldmanager/
+		- %currentWorkingDirectory%/vendor/campaignmonitor/
+		- %currentWorkingDirectory%/../../../wp-includes/
```

Comment:
We are! So that phpstan knows about types for functions, classes, methods, etc. in WP core

Reply:
This makes me wondering how phpstan is integrated with composer and if we should be doing it a different way, because it seems like phpstan should be able to just scan composer autoloaded classes for stuff like this but I am basing this on my instinct about how I would want this to work, which may be wrong. I'm wondering  if we are missing something in the phpstan.neon file like to provide type information for WordPress. Again, not really sure

```yaml
includes:
    - vendor/szepeviktor/phpstan-wordpress/extension.neon
```

From: https://github.com/szepeviktor/phpstan-wordpress

Reply:
I totally agree. I was looking at the config for that `phpstan-wordpress` plugin and I was surprised and confused about needing to manually include `wp-includes`.